config/routes.rb

```diff
@@ -6,14 +6,7 @@
   end
 
   if Settings.features.streaming_media
-    # download file_name must NOT include format extension (e.g.  .mp4)
-    # so insist on . being URL encoded (see media routing specs)
-    constraints file_name: %r{[^/\.]+} do
-      get '/media/:id/:file_name' => 'media#download', as: :media
-      get '/media/auth/:id/:file_name' => 'webauth#login_media_download', as: :auth_media_download
-    end
-
-    # stream file_name must include format extension, eg .../oo000oo0000.mp4/stream
+    # stream file_name must include format extension, eg .../oo000oo0000.mp4/verify_token
     #  other dots do not need to be URL encoded (see media routing specs)
```

Comment:
This comment made me think.  The specs that were checking for special chars have been removed.  (see https://github.com/sul-dlss/digital_stacks_rails/commit/287304de5adb0f5190fce79f2b1c4d97d9b711e7)   I wonder if we know what we need for stream name encoding for verify_token, and if we should have routing tests to illustrate it.  Or if we should remove this comment.   I also put a reference in sul-dlss/media#111